packages/status-js/src/client/member.ts

```diff
@@ -0,0 +1,17 @@
+import { generateUsername } from '../utils/generate-username'
+import { publicKeyToColorHash } from '../utils/public-key-to-color-hash'
+
+import type { ColorHash } from '../utils/public-key-to-color-hash'
+
+export class Member {
+  publicKey: string
+  username: string
+  colorHash: ColorHash
+
+  constructor(publicKey: string) {
+    this.publicKey = publicKey
+    this.username = generateUsername(publicKey)
+    // TODO: can it fail?
+    this.colorHash = publicKeyToColorHash(publicKey)!
```

Comment:
Not sure about `publicKeyToColorHash` returning `undefined`.

Reply:
Ditto https://github.com/status-im/status-web/pull/271#discussion_r895940611.